Result of `nixpkgs-review pr 107533` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>sourcetrail</li>
  </ul>
</details>

```
/private/tmp/nix-build-SourcetrailPythonIndexer-v1_db25_p5.drv-0/source/SourcetrailDB /private/tmp/nix-build-SourcetrailPythonIndexer-v1_db25_p5.drv-0/source
/private/tmp/nix-build-SourcetrailPythonIndexer-v1_db25_p5.drv-0/source
running tests
Traceback (most recent call last):
  File "/private/tmp/nix-build-SourcetrailPythonIndexer-v1_db25_p5.drv-0/source/SourcetrailDB/build/bindings_python/sourcetraildb.py", line 14, in swig_import_helper
    return importlib.import_module(mname)
  File "/nix/store/v3qi2n1cb9bj8h3076p3rz5fg0kpzz53-python3-3.8.6/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 973, in _find_and_load_unlocked
ModuleNotFoundError: No module named '_sourcetraildb'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "test_shallow.py", line 1, in <module>
    import shallow_indexer
  File "/private/tmp/nix-build-SourcetrailPythonIndexer-v1_db25_p5.drv-0/source/shallow_indexer.py", line 7, in <module>
    import sourcetraildb as srctrl
  File "/private/tmp/nix-build-SourcetrailPythonIndexer-v1_db25_p5.drv-0/source/SourcetrailDB/build/bindings_python/sourcetraildb.py", line 17, in <module>
    _sourcetraildb = swig_import_helper()
  File "/private/tmp/nix-build-SourcetrailPythonIndexer-v1_db25_p5.drv-0/source/SourcetrailDB/build/bindings_python/sourcetraildb.py", line 16, in swig_import_helper
    return importlib.import_module('_sourcetraildb')
  File "/nix/store/v3qi2n1cb9bj8h3076p3rz5fg0kpzz53-python3-3.8.6/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ModuleNotFoundError: No module named '_sourcetraildb'
builder for '/nix/store/jy59qs9qiwylkra31kx1xvrspxbhwm1i-SourcetrailPythonIndexer-v1_db25_p5.drv' failed with exit code 1
cannot build derivation '/nix/store/90cz10njry8m7gf13vcf9032r237bjg8-sourcetrail-2020.2.43.drv': 1 dependencies couldn't be built
error: build of '/nix/store/90cz10njry8m7gf13vcf9032r237bjg8-sourcetrail-2020.2.43.drv' failed
```

--- a/pkgs/development/tools/sourcetrail/python.nix
+++ b/pkgs/development/tools/sourcetrail/python.nix
@@ -64,5 +64,7 @@ stdenv.mkDerivation rec {
     description = "Python indexer for Sourcetrail";
     homepage = "https://github.com/CoatiSoftware/SourcetrailPythonIndexer";
     license = licenses.gpl3;
+    broken = stdenv.isDarwin;
+    # https://github.com/NixOS/nixpkgs/pull/107533#issuecomment-751063675
   };
 }

--- a/pkgs/development/tools/sourcetrail/python.nix
+++ b/pkgs/development/tools/sourcetrail/python.nix
@@ -8,7 +8,7 @@ stdenv.mkDerivation rec {
     owner = "CoatiSoftware";
     repo = pname;
     rev = version;
-    sha256 = "05hlpd3am029pv1wi6mys3q0ggp64axmg8bdf1fabl9cl9jffscq";
+    sha256 = "01jaigxigq6dvfwq018gn9qw7i6p4jm0y71lqzschfv9vwf6ga45";
     fetchSubmodules = true;
   };
 

